SignalServiceKit/src/Contacts/Threads/TSContactThread.h

```diff
@@ -22,6 +22,11 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (NSString *)contactIdFromThreadId:(NSString *)threadId;
 
+// This is only exposed for tests.
```

Comment:
There's two ways we tend to expose private functionality or state to tests:

* Through categories defined in the tests.
* Through DEBUG-only methods/properties on the class itself.

Previously I preferred the former since it kept the production code clean and neat.  Lately I prefer the latter because we're more likely to detect breakage (through build warnings or errors) even if don't build/run the tests.  

WDYT?

Reply:
I'm not sure I'm following... can we discuss offline?

Reply:
Yup, happy to.